Source/lighting.cpp

```diff
@@ -941,7 +942,8 @@ void AddUnLight(int i)
 		return;
 	}
 
-	Lights[i]._ldel = true;
+	auto &lights = Lights[i];
+	lights._ldel = true;

```

Comment:
If it's just a single use there is no need to place it in another variable.
```suggestion
	Lights[i]._ldel = true;
```